package.json

```diff
@@ -27,35 +27,39 @@
   },
   "dependencies": {
     "@enact/dev-utils": "^7.0.0-alpha.1",
-    "@storybook/addon-actions": "^8.2.4",
-    "@storybook/addon-controls": "^8.2.4",
-    "@storybook/addon-toolbars": "^8.2.4",
-    "babel-preset-enact": "^0.1.7",
+    "@storybook/addon-actions": "^8.2.9",
+    "@storybook/addon-controls": "^8.2.9",
+    "@storybook/addon-toolbars": "^8.2.9",
+    "babel-preset-enact": "^0.1.8",
     "css-loader": "^7.1.2",
     "less": "^4.2.0",
     "less-loader": "^12.2.0",
-    "mini-css-extract-plugin": "^2.9.0",
+    "mini-css-extract-plugin": "^2.9.1",
     "node-polyfill-webpack-plugin": "^3.0.0",
-    "postcss": "^8.4.38",
+    "postcss": "^8.4.45",
     "postcss-flexbugs-fixes": "^5.0.2",
     "postcss-global-import": "^1.0.6",
     "postcss-loader": "^8.1.1",
     "postcss-normalize": "^10.0.1",
-    "postcss-preset-env": "^9.5.14",
-    "postcss-resolution-independence": "^1.1.5",
-    "sass": "^1.69.5",
-    "sass-loader": "^13.3.2"
+    "postcss-preset-env": "^9.6.0",
+    "postcss-resolution-independence": "^1.1.6",
+    "sass": "^1.78.0",
+    "sass-loader": "^13.3.3"
   },
   "devDependencies": {
-    "@storybook/react": "^8.2.4",
-    "@storybook/react-webpack5": "^8.2.4",
+    "@storybook/react": "^8.2.9",
+    "@storybook/react-webpack5": "^8.2.9",
     "babel-loader": "^9.1.3",
     "eslint": "^8.57.0",
     "eslint-config-enact": "^5.0.0-alpha.1",
     "eslint-config-prettier": "^9.1.0",
-    "eslint-plugin-import": "^2.29.1",
-    "eslint-plugin-prettier": "^5.1.3",
-    "prettier": "^3.3.2",
-    "webpack": "^5.92.1"
+    "eslint-plugin-import": "^2.30.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "prettier": "^3.3.3",
+    "webpack": "^5.94.0"
+  },
+  "overrides": {
+    "postcss": "^8.4.45",
+    "send": "^0.19.0"
```

Comment:
@paul-beldean-lgp , please redo an update of minor versions, this should be solved using latest storybook versions